frontend/src/pages/main/payment/components/confirm/PaymentConfirmation.tsx

```diff
@@ -76,9 +74,8 @@ const PaymentConfirmation: React.FC<ConfirmProps> = ({qrCodeData}) => {
             <CardContent className="card-confirm-content">
                 <TextContent text={Content.INFORMATION}/>
```

Comment:
Since this is a title, I think you need to use bigger font here.

Reply:
I see you use this title for both Card titles and just text. It's incorrect. Create separate component with h6 for Card titles like here and Transactions in chart